doc/source/whatsnew/v0.22.0.txt

```diff
@@ -27,6 +27,27 @@ Other Enhancements
 - :class:`pandas.io.formats.style.Styler` now has method ``hide_index()`` to determine whether the index will be rendered in ouptut (:issue:`14194`)
 - :class:`pandas.io.formats.style.Styler` now has method ``hide_columns()`` to determine whether columns will be hidden in output (:issue:`14194`)
 
+``get_dummies`` now supports ``dtype`` argument
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:func:`get_dummies` function now accepts ``dtype`` argument, which forces specific dtype for new columns. When ``dtype`` is not specified or equals to ``None``, new columns will have dtype ``uint8`` (as before), so this change is backwards compatible. (:issue:`18330`)
```

Comment:
- Can remove `function` after `get_dummies`.
- now accepts *a* dtype argument
- replace *forces* with *specifies a*

Replace the second sentence with

    When ``dtype`` is not specified, the dtype will be ``uint8`` as before.